eZ/Bundle/EzPublishCoreBundle/DependencyInjection/Configuration/SiteAccessAware/Contextualizer.php

```diff
@@ -298,4 +298,21 @@ public function getGroupsBySiteAccess()
     {
         return $this->groupsBySiteAccess;
     }
+
+    private function array_merge_unshift(...$args)
+    {
+        $array = array();
+        foreach ($args as $arg) {
+            foreach ($arg as $key => $value) {
+                if (is_string($key)) {
```

Comment:
How the key could be anything but a string?

Reply:
numerical keys?

Reply:
Example would be [eng-GB] and [fre-CA].  These have numerical keys and we want when merged the site access to be first so we end up with [fre-CA,eng-GB]